docker/alpine-latest-openssl1.1-compat/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:lts-alpine
```

Comment:
Not sure if we want to pin to `lts-alpine3.17`

Reply:
Not sure either tbh

Reply:
also latest could be different than lts, the naming might be confusing

Reply:
On the other hand `openssl1.1-compat` might be gone in some future version of Alpine and it doesn't matter to us since we'll be using OpenSSL 3 anyway, so  not sure. Maybe let's pin it, we have a separate test where we don't pin the version.